llvm/include/llvm/ADT/SmallPtrSet.h

```diff
@@ -216,6 +216,20 @@ class SmallPtrSetImplBase : public DebugEpochBase {
     return EndPointer();
   }
 
+  bool contains_imp(const void *Ptr) const {
+    if (isSmall()) {
+      // Linear search for the item.
+      for (const void *const *APtr = SmallArray, *const *E =
+                                                     SmallArray + NumNonEmpty;
+           APtr != E; ++APtr)
```

Comment:
nit: the default formatting makes it look a bit weird, consider defining `APtr` and `E` outside of the loop